src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/ExportProviderCache.cs

```diff
@@ -221,7 +221,7 @@ ExportProvider IExportProviderFactory.CreateExportProvider()
                 }
             }
 
-            private void RequireForSingleExportProvider([DoesNotReturnIf(false)] bool condition)
```

Comment:
![BunniesWhatGIF](https://user-images.githubusercontent.com/475144/160204997-67fa5546-359d-491a-be6c-93afcafb3e81.gif)

Reply:
Oh ya woops, forgot to mention why this changed. I moved away from the hardcoded Nullable C# files in favor of sharing a singular one at the shared root of everything. This Test.Common class somehow has the actual nullable types available to it whereas all other product code does not. Meaning, I get ambiguous errors among the two specifically for this. Found that we don't actually create any warnings or errors in consuming call sites and that paired with it not being public I just figured it'd be easier to remove then spend the time teasing everything apart more.